src/hooks/useHistorical.js

```diff
@@ -3,16 +3,20 @@ import { useState } from 'react';
 import { filterItems } from '../helpers/filterItems';
 import { getHistorical } from '../services/getHistorical';
 
-const useHistorical = (contractAdress, chainId) => {
+const useHistorical = (
+  contractAdress,
+  chainId,
+  currentCoin,
+) => {
   const [historical, setHistorical] = useState([]);
+  console.log(currentCoin);
```

Comment:
Hi Maciel,

Can you delete this code line? Remember that we don't must have any console.log() in our code.